test/e2e/performanceprofile/functests/1_performance/irqbalance.go

```diff
@@ -71,6 +71,7 @@ var _ = Describe("[performance] Checking IRQBalance settings", Ordered, func() {
 	})
 
 	Context("Verify GloballyDisableIrqLoadBalancing Spec field", Label(string(label.Tier0)), func() {
+		testutils.KnownIssueJira("RHEL-79032")
```

Comment:
This will not work against RHEL bugs since the field check is for OCP bugs , see https://github.com/openshift/cluster-node-tuning-operator/blob/33b83c7ca017d81743847d43d35f2496f4dd0ab0/test/e2e/performanceprofile/functests/utils/utils.go#L67-L69 

So we either need to add a suitable check for RHEL bugs or for this specific case open a new OCP bug that will be linked to the RHEL one

Reply:
The functions seems to be working for RHEL bugs as well

Reply:
Hmm the `Fields.Status.Name == Planning` so it should be skipping but it's not. we need to look deeper to understand why

Reply:
`failed to retrieve status of Jira issue RHEL-79032: failed to get jira status of RHEL-79032: 401 401 Unauthorized`